### PR TITLE
add quick start with dataGrip doc

### PR DESCRIPTION
**Pull request by zwangsheng**
zwangsheng [Closes #422](https://github.com/yaooqinn/kyuubi/pull/422)    Target Issue Test Plan [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _Why are the changes needed?_

add quick start with dataGrip doc

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [√ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docs/quick_start/quick_start_with_datagrip.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+## 5.4 Interacting With Kyuubi server
+Now, you can interacing with kyuubi server
+
+The left side of the photo is the table, and the right side of the photo is the console.
+
+You can interact through the visual interface or code
+
+![workspace](../imgs/dataGrip/workspace.png)
+## 5.5 Epilogue
+There are many other amazing features in both Kyuubi and DBeaver and here is just the tip of the iceberg. The rest is for you to discover.
```
Comment (yaooqinn) [inline docs/quick_start/quick_start_with_datagrip.md:40]: nit: `DBeaver`  -> `dataGrip`

File: docs/quick_start/quick_start_with_datagrip.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# 5.Getting Started With Kyuubi and DataGrip
```
Comment (yaooqinn) [inline docs/quick_start/quick_start_with_datagrip.md:1]: the numbers here and below are unnecessary

File: docs/quick_start/quick_start_with_datagrip.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+## 5.1 what is DataGrip
```
Comment (yaooqinn) [inline docs/quick_start/quick_start_with_datagrip.md:2]: What

File: docs/quick_start/quick_start_with_datagrip.md (hunk 2 above)
Comment (zwangsheng) [reply docs/quick_start/quick_start_with_datagrip.md:1]: Borrowing from DBeaver，like DBeaver is 2 of quickStart and dataGrip is 5。

File: docs/quick_start/quick_start_with_datagrip.md (hunk 1 above)
Comment (zwangsheng) [reply docs/quick_start/quick_start_with_datagrip.md:40]: ok,, i will fix

File: docs/quick_start/quick_start_with_datagrip.md (hunk 3 above)
Comment (zwangsheng) [reply docs/quick_start/quick_start_with_datagrip.md:2]: ok,i will fix

File: docs/quick_start/quick_start_with_datagrip.md (hunk 2 above)
Comment (yaooqinn) [reply docs/quick_start/quick_start_with_datagrip.md:1]: They are auto generated

File: docs/quick_start/quick_start_with_datagrip.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+For debugging purpose, you can use ```tail -f``` or ```tailf``` to track the server log.
+## 5.3 Configurations
+### 5.3.1 Start DataGrip
+After you install DataGrip, just launch it.
+### 5.3.2 Select Database
+Substantially, this step is to choose a JDBC Driver type to use later. We can choose Apache Hive to set up a driver for Kyuubi.
+
+![select database](../imgs/dataGrip/select_database.png)
+### 5.3.3 Datasoure Driver
+You should first download missing driver files, just click on the link below, DataGrip will download and install those.
```
Comment (yaooqinn) [inline docs/quick_start/quick_start_with_datagrip.md:21]: You should first download the missing driver files. Just click on the link below, DataGrip will download and install those.

File: docs/quick_start/quick_start_with_datagrip.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+![datasource and driver](../imgs/dataGrip/datasource_and_driver.png)
+### 5.3.4 Generic JDBC Connection Settings
+After install dirver, you should configure the right host and port which you can find in kyuubi server log.By default, we use ```localhost``` and ```10009``` to config.
```
Comment (yaooqinn) [inline docs/quick_start/quick_start_with_datagrip.md:24]: dirver  -> driver

Comment (yaooqinn) [inline docs/quick_start/quick_start_with_datagrip.md:24]: log.By -> log. By

File: docs/quick_start/quick_start_with_datagrip.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+And alse, you can fill other config.
+
+After generic config, you can use test connection to test.
+
+![configuration](../imgs/dataGrip/configuration.png)
+## 5.4 Interacting With Kyuubi server
+Now, you can interacing with kyuubi server
```
Comment (yaooqinn) [inline docs/quick_start/quick_start_with_datagrip.md:32]: Now, you can interact with the Kyuubi server

File: docs/quick_start/quick_start_with_datagrip.md (hunk 5 above)
Comment (zwangsheng) [reply docs/quick_start/quick_start_with_datagrip.md:24]: ok， i will check all the spell first XD

File: docs/quick_start/quick_start_with_datagrip.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+The left side of the photo is the table, and the right side of the photo is the console.
+
+You can interact through the visual interface or code
```
Comment (yaooqinn) [inline docs/quick_start/quick_start_with_datagrip.md:36]: add . in the end

File: docs/quick_start/quick_start_with_datagrip.md (hunk 5 above)
Comment (yaooqinn) [inline docs/quick_start/quick_start_with_datagrip.md:24]: config -> configure

File: docs/quick_start/quick_start_with_datagrip.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+Substantially, this step is to choose a JDBC Driver type to use later. We can choose Apache Hive to set up a driver for Kyuubi.
+
+![select database](../imgs/dataGrip/select_database.png)
+### 5.3.3 Datasoure Driver
+You should first download missing driver files, just click on the link below, DataGrip will download and install those.
+![datasource and driver](../imgs/dataGrip/datasource_and_driver.png)
+### 5.3.4 Generic JDBC Connection Settings
+After install dirver, you should configure the right host and port which you can find in kyuubi server log.By default, we use ```localhost``` and ```10009``` to config.
+
+And alse, you can fill other config.
```
Comment (yaooqinn) [inline docs/quick_start/quick_start_with_datagrip.md:26]: also

Comment (yaooqinn) [inline docs/quick_start/quick_start_with_datagrip.md:26]: configs

File: docs/quick_start/quick_start_with_datagrip.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
+# 5.Getting Started With Kyuubi and DataGrip
+## 5.1 what is DataGrip
+DataGrip is a multi-engine database environment released by JetBrains, supporting MySQL and PostgreSQL, Microsoft SQL Server and Oracle, Sybase, DB2, SQLite, HyperSQL, Apache Derby and H2.
```
Comment (yaooqinn) [inline docs/quick_start/quick_start_with_datagrip.md:3]: , and H2.

**Comment by yaooqinn** [comment]
let's try to make the pictures in the same size, and you can follow https://kyuubi.readthedocs.io/en/latest/tools/build_document.html to verify locally first to see if it looks nice

**Comment by yaooqinn** [comment]
Thanks for your first contribution and welcome! Just left some minor comments. also cc @pan3793 @ulysses-you

File: docs/quick_start/quick_start_with_datagrip.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+![configuration](../imgs/dataGrip/configuration.png)
+## 5.4 Interacting With Kyuubi server
+Now, you can interacing with kyuubi server
+
+The left side of the photo is the table, and the right side of the photo is the console.
+
+You can interact through the visual interface or code
+
+![workspace](../imgs/dataGrip/workspace.png)
+## 5.5 Epilogue
```
Comment (ulysses-you) [inline docs/quick_start/quick_start_with_datagrip.md:39]: I believe `At the end` is more friendly .. since the word `Epilogue` is not used commonly.

**Review by yaooqinn — APPROVED** [review]
